src/js/variant-widget/eva-variant-widget.js

```diff
@@ -362,10 +362,16 @@ EvaVariantWidget.prototype = {
                     id: 'variant-grid-view-column',
                     xtype: 'templatecolumn',
                     tpl: new Ext.XTemplate('<a href="?variant={chromosome}:{start}:{reference:htmlEncode}:{alternate:htmlEncode}&species={[this.getSpecies(values)]}&annotationVersion={[this.getAnnotationVersion()]}" target="_blank" class="image-link"><img class="eva-grid-img-active" src="img/eva_logo.png"/></a>' +
-                        '&nbsp;<tpl if="this.getURL(values)"><a href="{[this.getURL(values)]}" class="image-link dbsnp_link" target="_blank" onclick="ga(\'send\', \'event\', { eventCategory: \'Variant Browser\', eventAction: \'dbSNP Link\', eventLabel:this})"><span>dbSNP</span></a>' +
-                        '<tpl else><span  style="opacity:0.2" class="eva-grid-img-inactive ">dbSNP</span></tpl>',
+                        '&nbsp;<tpl if="this.getEnsemblURL(values)"><a href="{[this.getEnsemblURL(values)]}" class="image-link ensembl_link" target="_blank" onclick="ga(\'send\', \'event\', { eventCategory: \'Variant Browser\', eventAction: \'Ensembl Link\', eventLabel:this})"><img class="eva-grid-img-active" src="img/ensembl_logo.png"/></a>' +
+                        '<tpl else><img style="opacity:0.2" class="eva-grid-img-inactive src="img/ensembl_logo.png"/></span></tpl>' +
```

Comment:
I think the tag that was opened was **img** but the one that was closed was **span**. As a result, the disabled Ensembl logo doesn't actually show up for variants with no available Ensembl link. See below

![image](https://user-images.githubusercontent.com/25152678/43782433-44512212-9a57-11e8-8680-6bc6aa137b29.png)